### PR TITLE
Selected and focused state splitted.

### PR DESCRIPTION
**Pull request by SezginEge**
Associated Issue: #962

### Summary of Changes

* Focused and selected state splitted up.

### Screenshots/Videos (OPTIONAL)

states

### CODE REVIEW

**Comment by SezginEge** [comment]
@jasonLaster @helenvholmes For now, I applied changes just for light theme. What do you think? I will apply changes to other themes according to your opinions.

File: src/components/SecondaryPanes/Frames.css [hunk 1]
```diff
@@ -47,13 +47,23 @@
   margin-right: 1em;
 }
 
-.frames ul li:hover,
+.frames ul li:hover {
+  background-color: rgba(236, 236, 236,0.6);
```
Comment (clarkbw) [inline src/components/SecondaryPanes/Frames.css]: We could call this something like: `--theme-selectable-background-hover` which should indicate that is a select-able object.

File: src/components/SecondaryPanes/Frames.css [hunk 2]
```diff
@@ -47,13 +47,23 @@
...
+}
+
+:root.theme-light .frames ul li:hover .location {
+  color: #49444c;
```
Comment (clarkbw) [inline src/components/SecondaryPanes/Frames.css]: If we follow that pattern, this becomes something like `--theme-selectable-color-hover`

File: src/components/SecondaryPanes/Frames.css [hunk 3]
```diff
@@ -47,13 +47,23 @@
...
+}
+
 .frames ul li.selected {
   background-color: var(--theme-selection-background);
   color: white;
 }
 
-:root.theme-light .frames ul li:hover .location,
+.frames ul li.selected:focus {
+  background-color: #4676b7;
```
Comment (clarkbw) [inline src/components/SecondaryPanes/Frames.css]: And this is something like: `--theme-selection-focused-background`

**Comment by clarkbw** [comment]
So I like the light theme colors, we just need the `dark` and `firebug` themes.  And then we can move it all over to variables.

**Comment by SezginEge** [comment]
@clarkbw Thanks for feedback 👍 I will send a pr as soon as possible.

**Comment by jasonLaster** [comment]
Hey @SezginEge, @clarkbw. I just took a look and am favoring a more soft touch:

1. re-use the hover color, which has a very similar meaning... focus is on and we can select another frame
2. removing the outline as we don't need it here


here's  a patch so you can see what i mean:

```diff


diff --git a/src/components/SecondaryPanes/Frames.css b/src/components/SecondaryPanes/Frames.css
index 6cf455b..fe8b9b5 100644
--- a/src/components/SecondaryPanes/Frames.css
+++ b/src/components/SecondaryPanes/Frames.css
@@ -48,22 +48,23 @@
 }

 .frames ul li:hover {
-  background-color: rgba(236, 236, 236,0.6);
+  background-color: var(--theme-toolbar-background-alt);
 }

 :root.theme-light .frames ul li:hover .location {
   color: #49444c;
 }

+.frames ul li:focus {
+  background-color: var(--theme-toolbar-background-alt);
+  outline: none;
+}
+
 .frames ul li.selected {
   background-color: var(--theme-selection-background);
   color: white;
 }

-.frames ul li.selected:focus {
-  background-color: #4676b7;
-}
-
 :root.theme-light .frames ul li.selected .location,
 :root.theme-firebug .frames ul li:hover .location,
 :root.theme-firebug .frames ul li.selected .location,
```

Curious what you think. We should of course test on all themes, which is easy now with the settings pane.

**Comment by SezginEge** [comment]
I am agree with you about remove outline. :+1:

Today I am planning to work on changes and send a new demo gif.

**Comment by SezginEge** [comment]
@clarkbw @jasonLaster I applied some changes for dark/firebug theme.  

Light: 
light

Dark:
dark

Firebug:
firebug

If it looks good to you, I will create theme variables and use them.

**Comment by jasonLaster** [comment]
@SezginEge are you formatting non-selected focused frames?

**Comment by jasonLaster** [comment]
From a conversation offline:

|||
|---|---|
|`.selected` | doesn't change |
|`:hover` | doesn't change |
|`:focus` | same as hoverr |
| `.selected:focus` | slightly darker than `.selected` |

selected-states

**Comment by SezginEge** [comment]
Recent changes: 

|Light: |Dark:|Firebug|
|--|--|--|
|light|dark|firebug|

I think firebug theme needs UI tweak. I cant any difference between .frames ul li:nth-of-type(2n) background color and hover background color. What do you think?

**Comment by jasonLaster** [comment]
Thanks. In that case we should remove the background color for even or odd tires and use a light border to divide the frames

**Comment by jasonLaster** [comment]
Hmm, actually im not sure. We want to be consistent with the source tree. I can look at this a bit more in a couple hours

**Comment by SezginEge** [comment]
Hey @jasonLaster ! How's this going? Do you have any suggestion?

**Comment by jasonLaster** [comment]
I pushed a couple small tweaks to the branch to simplify how we're doing the colors. The tweaks, improve the outline and focus state, which is the smallest possible change that fixes the issue.

The colors you chose for selected state are great and I'd like to revisit them when we look at selected states for the source tree and command bar.

While testing I noticed that we don't select the frame when you press `<enter>` on on the frame which would really help the accessibility of the frames component. I'll add an issue for this soon.

**Comment by SezginEge** [comment]
@jasonLaster @wldcordeiro Thank you for your time and feedback.

**Pull request merged**